### PR TITLE
Change tokio-tls and native-tls to 0.2…

### PR DESCRIPTION
**Pull request by jamesray1**
…in order to use the latest available version of open-ssl.

### CODE REVIEW

**Comment by vi** [comment]
What is the difference between `version = "0.2.0"` and `version = "0.2"`?

Maybe you are confusing `version = "0.2.0"` with `version = "=0.2.0"`?

**Comment by vi** [comment]
Isn't `0.2.0`, `0.2`, `^0.2.0` and `^0.2` all the same?

**Comment by jamesray1** [comment]
Yeah sorry my understanding wasn't great, just read https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html.

File: Cargo.toml [hunk 1]
```diff
@@ -27,9 +27,9 @@ base64 = "0.9"
 futures = { version = "0.1", optional = true }
 tokio-core = { version = "0.1", optional = true }
 tokio-io = { version = "^0.1.2", optional = true }
-tokio-tls = { version = "0.2.0", optional = true }
+tokio-tls = { version = "0.2.0 optional = true }
```
Comment (vi) [inline Cargo.toml]: What's wrong with the comma?

File: Cargo.toml [hunk 2]
```diff
@@ -27,9 +27,9 @@ base64 = "0.9"
...
 bytes = { version = "0.4", optional = true }
-native-tls = { version = "0.2.1", optional = true }
+native-tls = { version = "0.2.2" optional = true }
```
Comment (vi) [inline Cargo.toml]: What's wrong with using `native-tls` `0.2.1`? Does it fail to build or run unless it's `0.2.2`?

Note: `0.2.1` version requirement accepts not only `0.2.1`, but also `0.2.2`, `0.2.3` and so on, but not `0.3.0`.

**Comment by jamesray1** [comment]
The intention of this PR was to try to prevent a build error, but that is caused by another crate/dependency as detailed in https://github.com/paritytech/shasper/issues/19.

**Pull request closed without merging**